contracts/VaultV3.vy

```diff
@@ -907,6 +893,12 @@ def set_accountant(new_accountant: address):
     self.accountant = new_accountant
```

Comment:
I believe it was discussed to not allow the accounting manager to set the accountant since it is a high risk contract right? Should it be changed to the role manager

Reply:
Is there any downside to adding a bunch more roles? What about having one role per setter. eg

Set_Deposit_Limit_Role
Set_Accountant_Role
Revoke_Strategy_Role